data/json/effects_on_condition/melee_eocs.json

```diff
@@ -0,0 +1,988 @@
+                "then": [ { "math": [ "u_prof_level", "=", "3" ] }, { "math": [ "u_prof_lowest", "=", "min(u_prof_level, u_prof_lowest)" ] } ],
+                "else": [
+                  { "math": [ "u_val('proficiency', 'proficiency_id: prof_claws_master', 'format: 6400')", "+=", "rand(4)" ] },
+                  { "math": [ "u_prof_level", "=", "2" ] },
+                  { "math": [ "u_prof_lowest", "=", "min(u_prof_level, u_prof_lowest)" ] }
+                ]
+              }
+            ],
+            "else": [
+              { "math": [ "u_val('proficiency', 'proficiency_id: prof_claws_pro', 'format: 3200')", "+=", "rand(4)" ] },
+              { "math": [ "u_prof_level", "=", "1" ] },
+              { "math": [ "u_prof_lowest", "=", "min(u_prof_level, u_prof_lowest)" ] }
+            ]
+          }
+        ],
+        "else": [
+          { "math": [ "u_val('proficiency', 'proficiency_id: prof_claws_familiar', 'format: 1600')", "+=", "rand(4)" ] },
+          { "math": [ "u_prof_level", "=", "0" ] },
+          { "math": [ "u_prof_lowest", "=", "min(u_prof_level, u_prof_lowest)" ] }
+        ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "IS_SHIVS",
+    "global": false,
+    "condition": { "u_has_wielded_with_weapon_category": "SHIVS" },
+    "effect": [
+      {
+        "if": { "math": [ "u_val('proficiency', 'proficiency_id: prof_shivs_familiar', 'format: percent')", ">=", "100" ] },
+        "then": [
+          {
+            "if": { "math": [ "u_val('proficiency', 'proficiency_id: prof_shivs_pro', 'format: percent')", ">=", "100" ] },
+            "then": [
+              {
+                "if": { "math": [ "u_val('proficiency', 'proficiency_id: prof_shivs_master', 'format: percent')", ">=", "100" ] },
+                "then": [ { "math": [ "u_prof_level", "=", "3" ] }, { "math": [ "u_prof_lowest", "=", "min(u_prof_level, u_prof_lowest)" ] } ],
+                "else": [
+                  { "math": [ "u_val('proficiency', 'proficiency_id: prof_shivs_master', 'format: 6400')", "+=", "rand(4)" ] },
+                  { "math": [ "u_prof_level", "=", "2" ] },
+                  { "math": [ "u_prof_lowest", "=", "min(u_prof_level, u_prof_lowest)" ] }
+                ]
+              }
+            ],
+            "else": [
+              { "math": [ "u_val('proficiency', 'proficiency_id: prof_shivs_pro', 'format: 3200')", "+=", "rand(4)" ] },
+              { "math": [ "u_prof_level", "=", "1" ] },
+              { "math": [ "u_prof_lowest", "=", "min(u_prof_level, u_prof_lowest)" ] }
+            ]
+          }
+        ],
+        "else": [
+          { "math": [ "u_val('proficiency', 'proficiency_id: prof_shivs_familiar', 'format: 1600')", "+=", "rand(4)" ] },
+          { "math": [ "u_prof_level", "=", "0" ] },
+          { "math": [ "u_prof_lowest", "=", "min(u_prof_level, u_prof_lowest)" ] }
+        ]
+      }
+    ]
+  },
```

Comment:
This part of the code makes reference to the shivs proficiencies, even though they currently do not exist.

Reply:
If you do decide to add the proficiencies, some practice recipes for them would be cool too >_>.

Reply:
...man. Good catch, I completely spaced on that!

Reply:
Me too! I only noticed because I'm making a PR based on this one and noticed that there is no shivs proficiency, since the category is one that is not commonly used (As it should, they are weapons of necessity) I completely forgot about it hahaha.